client/src/hooks/index.js

```diff
@@ -32,3 +32,13 @@ export { default as useGrabCurrentTextSection } from "./texts/use-grab-current-t
 export { default as useCollapseContext } from "./contexts/use-collapse-context";
 
 export { default as useCurrentUser } from "./user/use-current-user";
+export { default as useSelectSettings } from "./settings/use-select-settings";
+export { default as useSelectAllIssues } from "./journals/use-select-all-issues";
+export { default as usePaginationState } from "./pagination/use-pagination-state";
+export { default as useFilterState } from "./filters/use-filter-state";
+export { default as useDispatchAllIssues } from "./journals/use-dispatch-all-issues";
+export { default as useSelectAllJournals } from "./journals/use-select-all-journals";
+export { default as useSelectJournal } from "./journals/use-select-journal";
+export { default as useSelectVolume } from "./journals/use-select-volume";
+export { default as useUrlFromState } from "./url/use-url-from-state";
+export { default as useDispatchAllJournals } from "./journals/use-dispatch-all-journals";
```

Comment:
This file is really growing now, so let's keep it easier to parse by grouping these hooks like the ones above. I'd be fine with putting an index file in each subfolder of `hooks` too if you think that's better.